### PR TITLE
AUTO-861 Add debug info to end of message if debug = true passed in

### PR DESCRIPTION
**Pull request by jchryssanthacopoulos**
The idea behind these changes is that users can detect LLM hallucinations and generally get more visibility into what the AI did.

**Test Case 1**

image

**Test Case 2**

image

**Test Case 3**

A "real" tour scheduled:

image

**Test Case 4**

In this case, the LLM hallucinates because it says a tour was scheduled but the action is not returned:

image

### CODE REVIEW

File: VLA_LLM/state.py [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+    def update_actions(self, action: str):
+        """Update state with given action.
+
+        Args:
+            action: Action to update state with
+
+        """
+        num_prospect_messages = str(self.agent_state[self.NUM_PROSPECT_MESSAGES])
+        current_actions = self.agent_state[self.ACTIONS].get(num_prospect_messages)
```
Comment (maccmux) [inline VLA_LLM/state.py]: Minor, could do
`current_actions = self.get_current_actions()`

File: main.py [hunk 2]
```diff
@@ -68,45 +67,39 @@ async def query_virtual_agent(inputs: QueryVLAInputs):
...
-
-    # create memory
-    messages = []
-    for interaction in conversation_history:
-        messages.extend([
-            HumanMessage(content=interaction['human_message']),
-            AIMessage(content=interaction['ai_message'])
-        ])
+    # get conversation history
+    messages = State(inputs.community_id, inputs.client_id).get_memory_of_messages()
```
Comment (maccmux) [inline main.py:71]: Is this used anywhere?

Comment (jchryssanthacopoulos) [reply main.py:71]: Yes, it's passed into the agent down below.

File: VLA_LLM/state.py (hunk 1 above)
Comment (jchryssanthacopoulos) [reply VLA_LLM/state.py]: Made the change.

**Pull request merged**